.pre-commit-config.yaml

```diff
@@ -50,7 +50,7 @@ repos:
     rev: 22.3.0
     hooks:
       - id: black
-        language_version: python3.9
+        language_version: python3.8
```

Comment:
I changed `language_version` to `python3.8`. Otherwise `pre-commit` would throw an error. 
Alternatively, pinning `python==3.9` in `environment.yml` and re-installing the conda environment worked. 

If this is a local problem, we can change `language_version` back to `python3.9`. Locally, I have to do one of the above before I can commit something.

Reply:
I think we can move on to 3.9 in environment.yml.